system_tests/pubsub.py

```diff
@@ -135,7 +135,6 @@ def test_create_subscription_w_ack_deadline(self):
     def test_list_subscriptions(self):
         TOPIC_NAME = 'list-sub' + unique_resource_id('-')
         topic = Config.CLIENT.topic(TOPIC_NAME)
-        self.assertFalse(retry_unavailable(topic.exists)())
```

Comment:
I think because of [this](https://travis-ci.org/GoogleCloudPlatform/gcloud-python/builds/156042092#L866)?